charts/dragonfly/templates/scheduler/scheduler-configmap.yaml

```diff
@@ -44,11 +44,12 @@ data:
         port: {{ .Values.redis.master.service.ports.redis }}
         password: {{ .Values.redis.auth.password }}
         {{- else }}
-        addrs:
-        - {{ .Values.externalRedis.host }}:{{ .Values.externalRedis.port }}
-        host: {{ .Values.externalRedis.host }}
-        port: {{ .Values.externalRedis.port }}
+        addrs: {{ .Values.externalRedis.addrs }}
```

Comment:
addrs is an array, you should use 'toYaml'
{{ toYaml .Values.externalRedis.Addrs | indent 10 }}